Classes/Controllers/PBGitHistoryController.m

```diff
@@ -590,19 +590,20 @@ - (void)showCommitsFromTree:(id)sender
 - (void) checkoutFiles:(id)sender
 {
 	NSMutableArray *files = [NSMutableArray array];
-	for (NSString *filePath in [sender representedObject])
+	for (NSString *filePath in [sender representedObject]) {
```

Comment:
I guess you want a line break there

If we want to go down such a road, we should probably write down on the desired locations for spaces first and try to get the code into that shape. Is that worth the effort?